### DISCUSSION TITLE
Problems uploading labeled dataset as prior knowledge for larger dataset

### DISCUSSION THREAD

**Discussion by PvdAa** [asreview/asreview#1262] [Q&A] [answered]
Hello everyone,

We encounter the following problem. We have a sample of 250 articles which have been screened in Rayyan and labeled in Rayyan.

This labeled dataset will be used as prior knowledge to further screen ±10000 articles in ASReview.

However every time I try to upload this in ASReview I get two different error-codes. 

1. 't' expected after '"' Please try again. 
2. ' ' expected after '"' Please try again. 

I tried uploading the file with the labels automatically from Rayyan. And I tried exporting the dataset without labels and adding an extra collum "label_included" with 1's and 0's manually. Both methods won't work. Without adding labels there are no error-codes. 

I tried in Oracle mode as well as Simulation mode. 

Does someone know what I should do different? 

Kind Regards

**Comment by gimoAI** [comment]
Hi @PvdAa ,

What file format are you using to export the file from Rayyan? And how are you adding the "label_included" column manually?

**Comment by PvdAa** [comment]
thanks @gimoAI , 


I tried two: CSV (add extra column in Numbers/Excel and then exported to CSV again) and RIS (couldnt add manually)

**Reply by gimoAI** [reply]
@PvdAa Since the error only occurs when you try to add the labels, the error is likely due to changes in the file during the exporting from Excel/Numbers. Can you try to export from Rayyan as `.csv`. Then in Excel, go to the data tab -> from text/CSV -> import the `.csv`. Add a column with the labels and export as `.xlsx`. Import this in ASReview.

Let me know if this works for you

**Comment by PvdAa** [comment]
Yes, this did the trick.

Thank you for your fast response and help!

**Comment by gimoAI** [comment, marked as answer]
Steps to move your project with labels from Rayyan to ASReview:
1. Export from Rayyan as `.csv`
2. In Excel, go to the `data` tab -> `from text/CSV` -> import the `.csv` file
3. Add a column with the labels: make sure the column name is one of the predetermined column names specified [here](https://asreview.readthedocs.io/en/latest/data_format.html)
4. Export as `.xlsx`
5. Import the `.xlsx` file into ASReview
